app/models/import_file_upload.rb

```diff
@@ -14,7 +14,7 @@ def service_dialog_list
       {
         :id          => index,
         :name        => dialog["label"],
-        :status_icon => ActionController::Base.helpers.image_path("16/#{status_icon}.png"),
```

Comment:
This is not following the recommendation in #13792 
`the :status and :status_icon keys should be replaced with a boolean :exists and the UI can infer the status and icon from that -- as is done in the model today`